"pretty-hrtime" not listed as a dev dependency

<!-- Search terms you tried before logging this (so others can find this issue more easily) -->
**Search Terms:**
* pretty-hrtime
* pnpm

**Steps To Reproduce**

1. Delete `node_modules/` directory (if it exists) and install `pnpm`.
2. `$ pnpm install`
3. `$ jake local`

**Expected behavior:**

```
node lib/tsc.js -b src
node built/local/tsc.js -b built/local/typescriptServices.tsconfig.json
node built/local/tsc.js -b built/local/tsserverlibrary.tsconfig.json
node lib/tsc.js -b scripts
node scripts/generateLocalizedDiagnosticMessages.js src/loc/lcl built/local src/compiler/diagnosticMessages.generated.json
```

**Actual behavior:**

```
jake aborted.
Error: Cannot find module 'pretty-hrtime'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
(See full trace by running task with --trace)
```

**Notes**
The issue is not present when using `npm` or `yarn` to install packages, because `pnpm` does not use a flat directory structure (please see [this](https://medium.com/pnpm/flat-node-modules-is-not-the-only-way-d2e40f7296a3)) and listing `pretty-hrtime` as a dev dependency of this project fixes the issue.

`pretty-hrtime` is currently being used [here](https://github.com/Microsoft/TypeScript/blob/111300c/scripts/build/project.js#L17).

--- a/package.json
+++ b/package.json
@@ -82,6 +82,7 @@
         "mocha": "latest",
         "mocha-fivemat-progress-reporter": "latest",
         "plugin-error": "latest",
+        "pretty-hrtime": "^1.0.3",
         "prex": "^0.4.3",
         "q": "latest",
         "remove-internal": "^2.9.2",

